.github/workflows/common.yml

```diff
@@ -79,6 +79,10 @@ jobs:
         shell: bash
         run: RUST_LOG=hyperon=debug cargo test
 
+      - name: Test Rust build with online feature
+        shell: bash
+        run: RUST_LOG=hyperon=debug cargo test --features "git online-test"
+
```

Comment:
This includes all tests which are not marked by `online-test` feature as well. Thus I suggest removing this and adding `--features "git online-test"`  to the upper step.
```suggestion
```